esp/src/src-react/hooks/Workunit.ts

```diff
@@ -18,6 +18,7 @@ export function useWorkunit(wuid: string, full: boolean = false): [Workunit, WUS
     const [lastUpdate, setLastUpdate] = React.useState(Date.now());
 
     React.useEffect(() => {
+        if (!wuid) return;
```

Comment:
Should probably return `[undefined, State.Unknown (or 999), lastUpdate]`
(999 means wu is in process of being deleted)

Reply:
I think the return of the useWorkunit hook around line 43 should be unaffected, this one should just be preventing the execution of the inner useEffect, attempting to attach and watch, if the wuid is undefined.

Reply:
Yes you are correct!